docs/cue/reference/components/sources/journald.cue

```diff
@@ -103,6 +103,17 @@ components: sources: journald: {
 				syntax: "literal"
 			}
 		}
+		journal_files_dir: {
+			common:      false
+			description: "The full path of the journal files directory."
```

Comment:
It's probably worth making a comment about the default here, something to indicate that if it is not set it will default to reading from the system journal.

I think naming this something like `journal_directory` might be a little more descriptive, using the terminology from the `journalctl` manual page.